Managers/FrameMan.h

```diff
@@ -551,8 +557,13 @@ namespace RTE {
 
 		int m_GfxDriver; //!< The graphics driver that will be used for rendering.
 
-		unsigned short m_ScreenResX; //!< Width of the physical screen (desktop resolution). 
-		unsigned short m_ScreenResY; //!< Height of the physical screen (desktop resolution).
+		bool m_DisableMultiScreenResolutionValidation; //!< Whether to disable resolution validation or not. Allows setting whatever crazy resolution that may or may not crash.
```

Comment:
Nitpick - I guess the method comment here should mention that it's for multiscreens, even though the variable name makes that super clear haha.